modules/install-sno-ibm-z-kvm.adoc

```diff
@@ -0,0 +1,173 @@
+----
++
+[source,terminal]
+----
+$ tar zxvf openshift-install-linux.tar.gz
+----
++
+[source,terminal]
+----
+$ chmod +x openshift-install
+----
+
+. Prepare the `install-config.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: <domain> <1>
+compute:
+- name: worker
+  replicas: 0 <2>
+controlPlane:
+  name: master
+  replicas: 1 <3>
+metadata:
+  name: <name> <4>
+networking: <5>
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16 <6>
+  networkType: OVNKubernetes
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  none: {}
+bootstrapInPlace:
+  installationDisk: /dev/disk/by-id/<disk_id> <7>
+pullSecret: '<pull_secret>' <8>
+sshKey: |
+  <ssh_key> <9>
+----
+<1> Add the cluster domain name.
+<2> Set the `compute` replicas to `0`. This makes the control plane node schedulable.
+<3> Set the `controlPlane` replicas to `1`. In conjunction with the previous `compute` setting, this setting ensures the cluster runs on a single node.
+<4> Set the `metadata` name to the cluster name.
+<5> Set the `networking` details. OVN-Kubernetes is the only allowed network plugin type for single-node clusters.
+<6> Set the `cidr` value to match the subnet of the {sno} cluster.
+<7> Set the path to the installation disk drive, for example, `/dev/disk/by-id/wwn-0x64cd98f04fde100024684cf3034da5c2`.
+<8> Copy the {cluster-manager-url-pull} and add the contents to this configuration setting.
+<9> Add the public SSH key from the administration host so that you can log in to the cluster after installation.
+
+. Generate {product-title} assets by running the following commands:
++
+[source,terminal]
+----
+$ mkdir ocp
+----
++
+[source,terminal]
+----
+$ cp install-config.yaml ocp
+----
++
+[source,terminal]
+----
+$ ./openshift-install --dir=ocp create single-node-ignition-config
+----
+
+. Obtain the {op-system-base} `kernel`, `initramfs`, and `rootfs` artifacts from the link:https://access.redhat.com/downloads/content/290[Product Downloads] page on the Red Hat Customer Portal or from the link:https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/latest/[{op-system} image mirror] page.
++
+[IMPORTANT]
+====
+The {op-system} images might not change with every release of {product-title}. You must download images with the highest version that is less than or equal to the {product-title} version that you install. Only use the appropriate `kernel`, `initramfs`, and `rootfs` artifacts described in the following procedure.
+====
++
+The file names contain the {product-title} version number. They resemble the following examples:
++
+`kernel`:: `rhcos-<version>-live-kernel-<architecture>`
+`initramfs`:: `rhcos-<version>-live-initramfs.<architecture>.img`
+`rootfs`:: `rhcos-<version>-live-rootfs.<architecture>.img`
++
+. Before you launch `virt-install`, move the following files and artifacts to an HTTP or HTTPS server:
+
```

Comment:
Nit: If you use a `+` here, you can just use a single `*` for the bullet to get the correct alignment. Not a big deal at all, just an FYI.